### PR TITLE
Update text for Developer preview and Clone existing PVC

### PR DESCRIPTION
**Pull request by hstastna**
## 📝 Description

Update the popovers' text for the _Developer preview_ label and for _Clone existing PVC_ checkbox in _Add volume to boot from_ modal window, according to the design updates.

## 🎥 Screenshots
**Before:**
 _Developer preview_ label:
dev_before
 _Clone existing PVC_ checkbox:
pvc_before

**After:**
dev_after
pvc_after

### CODE REVIEW

File: locales/en/plugin__kubevirt-plugin.json [hunk 1]
```diff
@@ -350,7 +350,7 @@
   "Details": "Details",
   "Detected file extension is {{fileNameExtension}}": "Detected file extension is {{fileNameExtension}}",
   "Developer preview": "Developer preview",
-  "Developer preview features are not intended to be used in production environments. The clusters deployed with the developer preview features are considered to be development clusters and are not supported through the Red Hat Customer Portal case management system.": "Developer preview features are not intended to be used in production environments. The clusters deployed with the developer preview features are considered to be development clusters and are not supported through the Red Hat Customer Portal case management system.",
+  "Developer preview features are not intended to use in production environments. The clusters deployed with the developer preview features are developmental clusters and are not currently supported by Red Hat.": "Developer preview features are not intended to use in production environments. The clusters deployed with the developer preview features are developmental clusters and are not currently supported by Red Hat.",
```
Comment (apinnick) [inline locales/en/plugin__kubevirt-plugin.json:353]: I would like this reviewed by docs because it is verbose. I can't tag Sabrina Jess, however.

Comment (hstastna) [reply locales/en/plugin__kubevirt-plugin.json:353]: Copy and send her the link for these comments.

Comment (sabrinajess) [reply locales/en/plugin__kubevirt-plugin.json:353]: Developer preview features are not currently supported by Red Hat and not intended for use in production environments.

**Pull request merged**